pkg/signature/kms/hashivault/client.go

```diff
@@ -94,6 +108,8 @@ func newHashivaultClient(keyResourceID string) (*hashivaultClient, error) {
 		return nil, errors.Wrap(err, "new vault client")
 	}
 
+	client.SetToken(token)
```

Comment:
just curious - why are we doing this now VS not before?

Reply:
hello @bobcallaway, by default, the Vault client is using the `VAULT_TOKEN` environment variable implicitly, so we just check the existence of the environment variable instead of using it explicitly, but now we are reading this from a file, so we have to set it explicitly.

Reply:
makes sense, thanks @developer-guy!